recipe/meta.yaml

```diff
@@ -1,17 +1,17 @@
-{% set version = "1.1.8" %}
+{% set version = "1.2.1" %}
```

Comment:
The latest release is https://github.com/scrapinghub/dateparser/releases/tag/v1.2.2. I think we should build v1.2.2